lib/splitclient-rb/validators.rb

```diff
@@ -38,6 +41,25 @@ def valid_matcher_arguments(args)
       true
     end
 
+    def valid_flag_sets(method, flag_sets)
```

Comment:
please use the same name defined in the spec. Maybe you can create a `FlagSetsValidator` file with this validations

Reply:
We agreed to make it technical debt